caravel/bin/caravel

```diff
@@ -125,6 +125,34 @@ def refresh_druid():
             "[" + cluster.cluster_name + "]")
     session.commit()
 
+@manager.option(
+    '-p', '--prefix', default='data_',
```

Comment:
I'd make the prefix not required and empty by default so one can insert all the tables of a database if needed.